.github/workflows/.windows.yml

```diff
@@ -258,6 +182,8 @@ jobs:
           - builtin
           - containerd
         test: ${{ fromJson(needs.integration-test-prepare.outputs.matrix) }}
+        exclude:
+          - runtime: ${{ inputs.use_snapshotter && 'builtin' || 'nosuchruntime' }}
```

Comment:
I probably could have just made the `runtime` list itself be affected by use_snapshotter, but I think this is clearer for readers anyway.

Reply:
Implemented (better) in #46572, no longer relevant here.